mobile/lib/domain/interfaces/sync_api.interface.dart

```diff
@@ -1,7 +1,9 @@
 import 'package:immich_mobile/domain/models/sync/sync_event.model.dart';
 
 abstract interface class ISyncApiRepository {
-  Future<void> ack(String data);
+  Future<void> ack(List<String> data);
 
   Stream<List<SyncEvent>> watchUserSyncEvent();
+
+  Stream<List<SyncEvent>> watchPartnerSyncEvent();
```

Comment:
We don't watch events IMO. Yes, it's a stream, but we're not watching the stream forever and it doesn't work like other usages of `watch*`. Can we call this `getUserEvents`?